packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -704,6 +740,7 @@ void main() {
           ),
         ),
       );
+      await tester.pump();
```

Comment:
Can you add a comment on why the pump is needed?
i.e. The Column's SelectionContainer takes one frame to register to SelectionArea.